alpaca/trading/client.py

```diff
@@ -317,6 +321,28 @@ def close_position(
 
         return Order(**response)
 
+    def exercise_option_contract(
+        self,
+        symbol_or_contract_id: Union[UUID, str],
+    ) -> None:
+        """
+        This endpoint enables users to exercise a held option contract, converting it into the underlying asset based on the specified terms.
+        All available held shares of this option contract will be exercised.
+        By default, Alpaca will automatically exercise in-the-money (ITM) contracts at expiry.
+        Exercise requests will be processed immediately once received. Exercise requests submitted outside market hours will be rejected.
+        To cancel an exercise request or to submit a Do-not-exercise (DNE) instruction, please contact our support team.
+
+        Args:
+            symbol_or_contract_id (Union[UUID, str]): Option contract symbol or ID.
+
+        Returns:
+            None
+        """
+        symbol_or_contract_id = validate_symbol_or_contract_id(symbol_or_contract_id)
+        response = self.post(
```

Comment:
I believe `response` is unused. Not sure if you wanted to log or something.

Reply:
Good catch! Please let me remove it.